### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nathankellenicki**
Made notes about `setcap` to enable elevated privileges for Noble clearer.

### CODE REVIEW

**Comment by rzr** [comment]
feedback welcome

File: README.md [hunk 1]
```diff
@@ -65,11 +65,13 @@ noble.on('discover', async (peripheral) => {
 #### Linux
 
  * Kernel version 3.6 or above
- * `libbluetooth-dev`
- * Run the following command to grant node the necessary privileges to read BLE data: `sudo setcap cap_net_raw+eip $(eval readlink -f $(which node))` ([Explanation](https://github.com/abandonware/noble#running-without-rootsudo-linux-specific))
+ * `libbluetooth-dev` needs to be installed. For instructions for specific distributions, see below.
+ * To set the necessarilly privileges to run without sudo, [see this section](https://github.com/abandonware/noble#running-without-rootsudo-linux-specific). This is required for all distributions (Raspbian, Ubuntu, Fedora, etc). You will not get any errors if running without sudo, but nothing will happen.
```
Comment (mrstegeman) [inline README.md:69]: Necessary, not necessarily

**Comment by rzr** [comment]
Please update this PR

**Comment by rzr** [comment]
Ping, or anyone can open a new PR that would preserve attribution ?

**Comment by atrovato** [comment]
@rzr this one is merged with #177 
We can close it.

**Pull request closed without merging**